### PR TITLE
Fix: Fixed issue where turning off grouping can show an empty view

### LINKED ISSUE #5011: Bug: Turning off grouping can show an empty view
**Explain the bug**

When grouping (implemented in #4826) is turned on and the user clicks on a header to show the group names only and then turns off grouping the view becomes empty.

**To reproduce**

1. Go to any folder
2. Click on Arrangement options
3. Select any grouping type (eg. Name)
4. Click on a group header
5. Click on Arrangement options again
6. Select None in the Group by options
5. See the error (empty view)

**Expected behavior**

The original folder view should be restored when Group by > None is being selected.

**Screenshots**

image image

**System Information**

 - OS Version: Windows 10 20H2 19042.985
 - App version: 2.3.11.0

**Additional comment**

This is not an issue when the folder contents were visible, not just the group headers.

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- Closes #5011 

**Validation**
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml [hunk 1]
```diff
@@ -147,6 +147,7 @@
 			Visibility="Collapsed" />
 
 		<SemanticZoom
+            x:Name="RootGridZoom"
```
Comment (yaichenbaum) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:150]: Is spacing off? Might want to change your Visual Studio settings to use tabs.

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/ColumnViewBase.xaml:150]: I'm sorry it seems to reset each time I open it. I'll reinstall it

Comment (ferrariofilippo) [reply src/Files.App/Views/LayoutModes/ColumnViewBase.xaml]: Tabs
Are these the right settings?

Comment (yaichenbaum) [reply src/Files.App/Views/LayoutModes/ColumnViewBase.xaml]: Yes

**Comment by d2dyno1** [comment]
It's hard to determine what changes were made if spacing is changed (tabs/spaces). I'd advise change the spacing in a separate PR because it can also quickly result in merge conflicts

**Comment by yaichenbaum** [comment]
@d2dyno1 you can hide whitespace changes
image

**Comment by d2dyno1** [comment]
@ferrariofilippo Could you recreate this PR without whitespace changes to avoid conflicts?

**Comment by yaichenbaum** [comment]
It's probably best to keep the pr open and just resolve conflicts.

**Comment by ferrariofilippo** [comment]
> It's hard to determine what changes were made if spacing is changed (tabs/spaces). I'd advise change the spacing in a separate PR because it can also quickly result in merge conflicts

I added the method `private void ZoomIn(object sender, GroupOption option)` in all layouts

**Comment by yaichenbaum** [comment]
Is that something that can be done in base layout?

**Comment by ferrariofilippo** [comment]
We can't since `ZoomIn` needs a control from the xaml (`RootGrid` name of `SemanticZoom` control)

**Comment by yaichenbaum** [comment]
@d2dyno1 the conflicts have been resolved, can you take a look again?

**Review by d2dyno1 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
Thank you

**Pull request merged**